webapp/.npmrc

```diff
@@ -1,2 +1,3 @@
 save-exact=true
 legacy-peer-deps=true
+global-style=true
```

Comment:
This is the main change. There may be some dedupe cost, but only this or `--legacy-bundling` (which disables all deduping) paired with reinstalling `@types/react-select@3.0.19` in the channels workspace seemed to fix the latest cross-workspace module resolution issues.

Reply:
Thanks for digging into this. I think this would've helped fix some other issues I've fought with as well. From what I can find, npm used to have this behaviour years ago then stopped doing it despite knowing there were were issues with it, and later re-added this as an option. I haven't found any docs suggesting this is at all a bad idea, and it works similarly in pnpm as well apparently.

That setting got replaced by `install-strategy=shallow` [in npm 9](https://github.blog/changelog/2022-10-24-npm-v9-0-0-released/) which we'll need to remember to change, but this sounds good to me.